doc/hardware/peripherals/i2c.rst

```diff
@@ -8,52 +8,48 @@ Overview
 
 .. note::
 
-   Zephyr recognizes the need to change the terms "master" and "slave"
-   used in the current `I2C Specification <i2c-specification>`_.  This
-   will be done when the conditions identified in
-   :ref:`coding_guideline_inclusive_language` have been met.  Existing
-   documentation, data structures, functions, and value symbols in code
-   are likely to change at that point.
+   The terminology used in the specification has changed. See
+   https://github.com/zephyrproject-rtos/zephyr/issues/27033
```

Comment:
I'd prefer a reference to the specification instead of a Zephyr issue. 

```suggestion
   The terminology used in the Zephyr I2C APIs follows that of the `NXP I2C-bus specification and user manual (UM10204), Rev. 7.0 <https://www.nxp.com/docs/en/user-guide/UM10204.pdf>`_. 
```